models/platform-limits/src/limit-with-current/generator.ts

```diff
@@ -0,0 +1,15 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TLimitWithCurrent } from './types';
+
+/**
+ * limit - Number
+ * current - Number
+ */
+const generator = Generator<TLimitWithCurrent>({
+  fields: {
+    limit: fake((f) => f.number.int(10)),
+    current: fake((f) => f.number.int(10)),
```

Comment:
Shouldn't `current` be a lower number than `limit` in the generator? (default value)

Reply:
They both can have same number. Refer above comment.

Reply:
Thanks for the explanation.

What I mean is that, if they both have the same number, the limit will be considered exceeded, right?
If that's correct, then the default value for this limit that the model will be returning will be in the exceeded state, where perhaps it makes more sense that the default value is an allowed one.

Would that make sense to you?

Reply:
I was thinking that by keeping the default state with same maximum value can generate all the different possible states (eg: not exceeded, with warning exceeded, exceeded). Since we already have a preset for all the different states, if user prefer only `allowed` one or `exceeded one` then they can choose the corresponding preset. 

Do you think we make default to generate only `not-exceeded` value? or both `not-exceeded` & `with-warning-exceeded` (because both can be considered as `allowed` one?
In other words, do you prefer the current is `int(9)` or `int(7)` or something else?

Reply:
My vote wold be for `int(7)` as the default one and users customizing that value or using a preset if they want something else.

Reply:
Done https://github.com/commercetools/test-data/pull/655/commits/aa7ad4e28b70b1fa5e0b1d52d4c63aaa99db10d1